model/vm_host.rb

```diff
@@ -251,14 +251,24 @@ def init_health_monitor_session
 
   def check_pulse(session:, previous_pulse:)
     reading = begin
-      session[:ssh_session].exec!("true")
-      "up"
-    rescue
+      all_bdevs = storage_devices.flat_map(&:blk_dev_serial_number)
+      commands = all_bdevs.map do |serial|
+        "sudo smartctl -H /dev/disk/by-id/$(ls /dev/disk/by-id/ | grep -E '#{serial}$') | grep -qE 'OK|PASSED' && echo 'up' || echo 'down'"
+      end
+      stdout = session[:ssh_session].exec!(commands.join("; "))
+      all_up = stdout.split("\n").all? { |res| res.strip == "up" }
+      all_up ? "up" : "down"
+    rescue => ex
+      Clog.emit(ex)
```

Comment:
I don't think that's how `Clog.emit` is used with exceptions. You banner it with a static string and then maybe use `Util.exception_to_hash(ex)`